src/overview/components/Overview.jsx

```diff
@@ -21,16 +21,17 @@ const Overview = props => (
         <Head />
         <Header {...props} />
 
-        <Tooltip
+        {/* <Tooltip
```

Comment:
This can be removed as well, like the filter button. And the other commented-out proptypes at the bottom of this module.
 
There are also a number of actions and selectors related to the tooltip state that are currently hooked up to the rest of the overview code and will still be firing off for no reason, even with this commented out. Probably want to track those down and unhook them from the overview components (press ctrl+shift+L to see the redux log and find the tooltip action name).